interface/billing/sl_eob_search.php

```diff
@@ -404,40 +404,59 @@ function upload_file_to_client_pdf($file_to_send, $aPatFirstName = '', $aPatID =
     //    notice  = 1 for first notice, 2 for second, etc.
     //    detail  = array of details, see invoice_summary.inc.php
     //
-    if ($stmt['cid'] != $row['pid']) {
-      if (!empty($stmt)) ++$stmt_count;
-      $stmt['cid'] = $row['pid'];
-      $stmt['pid'] = $row['pid'];
-      $stmt['dun_count'] = $row['stmt_count'];
-      $stmt['bill_note'] = $row['pat_billing_note'];
-      $stmt['enc_bill_note'] = $row['enc_billing_note'];
-      $stmt['bill_level'] = $row['last_level_billed'];
-      $stmt['level_closed'] = $row['last_level_closed'];
-      $stmt['patient'] = $row['fname'] . ' ' . $row['lname'];
-      $stmt['encounter'] = $row['encounter'];
+
+   if ($stmt['cid'] != $row['pid']) {
+        if (!empty($stmt)) {
+            ++$stmt_count;
+        }
+        if ($_POST['form_pdf']) {
+                fwrite($fhprint, make_statement($stmt));
+        }
+        if ($_POST['form_email']) {
+                fwrite($fhprint, create_html_statement($stmt));
+                upload_file_to_client_email($stmt['pid'], $STMT_TEMP_FILE);
+                fclose($fhprint);
+                sleep(1);
+                $fhprint = fopen($STMT_TEMP_FILE, 'w');
+                $stmt = array();
+        }
+
+      //if ($stmt['cid'] != $row['pid']) {
+        //if (!empty($stmt)) ++$stmt_count;
+        //fwrite($fhprint, create_pdf_statement($stmt));
+        $stmt['cid'] = $row['pid'];
+        $stmt['pid'] = $row['pid'];
+                $stmt['dun_count'] = $row['stmt_count'];
+                $stmt['bill_note'] = $row['pat_billing_note'];
+                $stmt['enc_bill_note'] = $row['enc_billing_note'];
+                $stmt['bill_note'] = $row['billing_note'];
```

Comment:
did you mean to add the line above here (note it overwrite the $stmt['bill_note'] 3 lines above this.